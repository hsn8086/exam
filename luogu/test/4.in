100 10
289219489 444975000 959321788 530472351 196624110 481057868 506447290 751633087 790427349 529535012 812111363 111035607 323165044 882143189 221673236 836702529 30857318 64360152 328968323 367854064 627487733 572551683 34399552 849445428 591856234 668028183 665242895 817979560 386480648 12605291 181139547 841278482 727313094 22783703 964419054 867788652 101961338 22278184 283608484 689019755 207021797 269782896 645755440 363320384 887824341 780016377 803603829 354872691 118574703 886548456 477733910 457539618 58343593 34691189 265015127 921845864 590319048 580505257 440295705 90246581 181568975 3585902 77739759 30176694 802312062 431463436 373077584 820604767 672926200 272312327 851729688 762745984 340428286 761508816 240215974 496746924 911435902 868862862 772549451 395821561 344709450 664860707 373090164 124553869 781751136 419580002 278477745 191765730 18875737 114471520 523961559 90650630 249603557 590728228 502870227 269350664 100106875 980664860 851850496 201075937
