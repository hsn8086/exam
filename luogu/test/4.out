867788652 868862862 882143189 886548456 887824341 911435902 921845864 959321788 964419054 980664860 